### PR TITLE
No Babel

### LINKED ISSUE #93: Require Node 4.4.0 and higher

### PR DESCRIPTION
**Pull request by misteroneill**
Fixes #93 and removes Babel entirely from the generator. There's no longer any need to build in order for code to pass videojs-standard.

### CODE REVIEW

File: bin/videojs-generator-cleanup.js [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
-var chalk = require('chalk');
-var fs = require('fs-extra');
-var _ = require('lodash');
-var path = require('path');
-var semver = require('semver');
-var sh = require('shelljs');
+'use strict';
 
-var configs = {
+/* eslint-env node */
```
Comment (BrandonOCasey) [inline bin/videojs-generator-cleanup.js]: isn't this line a part of videojs-standard already?

File: package.json [hunk 2]
```diff
@@ -14,28 +14,20 @@
...
-    "build:docs": "doctoc README.md docs/",
-    "build:generator": "babel --ignore templates/**/* src/app -d generators/app && npm run build:generator:copy",
-    "build:generator:copy": "node -e \"require('shelljs').cp('-r', 'src/app/templates', 'generators/app')\"",
-    "build:test": "babel src/test -d es5-test",
     "lint": "vjsstandard",
-    "prepublish": "npm run build",
-    "pretest": "npm run lint && npm run build",
-    "test": "mocha es5-test/*.test.js",
+    "pretest": "npm run lint",
+    "test": "mocha test/*.test.js",
```
Comment (BrandonOCasey) [inline package.json]: we may want to add back a prepublish for test/lint

File: generators/app/index.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-import path from 'path';
-import tsmlj from 'tsmlj';
-import yeoman from 'yeoman-generator';
-import yosay from 'yosay';
-import shelljs from 'shelljs';
-
-import {PREFIX} from './constants';
-import packageJSON from './package-json';
-import * as validators from './validators';
+'use strict';
```
Comment (BrandonOCasey) [inline generators/app/index.js]: I think all node modules have this on by default. So I don't think we need to specify this here or in the other files.

Comment (misteroneill) [inline generators/app/index.js]: Node 4.4 complains about using `const` and `let` outside of strict mode.

Comment (BrandonOCasey) [inline generators/app/index.js]: ok

**Comment by BrandonOCasey** [comment]
I think we want to add the node 4.4+ requirement to the package.json template file

**Pull request merged**